### PR TITLE
Update voovan to 4.2.3

### PR DESCRIPTION
**Pull request by helyho**
1. Update voovan to 4.2.3
2. fixed framework name in benchmark_config.json as your suggest

### CODE REVIEW

**Comment by helyho** [comment]
travis ci passed, please  merge this pr, thanks

File: frameworks/Java/voovan/benchmark_config.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "framework": "voovan",
   "tests": [{
-    "default": {
+    "mvc": {
```
Comment (nbrady-techempower) [inline frameworks/Java/voovan/benchmark_config.json:4]: This needs to be called default otherwise the toolset gives off lots of warnings

**Comment by helyho** [comment]
i fix this to "default"

File: frameworks/Java/voovan/benchmark_config.json (hunk 1 above)
Comment (helyho) [reply frameworks/Java/voovan/benchmark_config.json]: i fix this to "default", thanks

**Pull request merged**